setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = pytest-doctestplus
-url = https://github.com/astropy/pytest-doctestplus
-author = The Astropy Developers
+url = https://github.com/scientific-python/pytest-doctestplus
+author = Scientific Python
 author_email = astropy.team@gmail.com
```

Comment:
```suggestion
author_email = ci-infrastructure@discuss.scientific-python.org
```

This doesn't exist yet, it is just a suggestion. Let me know if I should create this mailing list or if you would prefer to use a different name or would prefer to keep it as is.

Reply:
Make no sense to still list Astropy email, so that email definitely has to be replaced but I don't have a preference with what... I don't even know who is checking that current `astropy.team` email.

Reply:
What about not having an email listed? The astropy one never worked for sure, at least it didn't get to the maintainers. Or maybe no one sent an email ever.

Reply:
As long as twine check doesn't complain, that is fine by me.

Reply:
why would it, is there a recent change so it stopped working with non-org repos? 
(I mean you've been the one who moved it into the repo last week, and a lot of orgs are still missing, so would be weird if things would stop working)

Reply:
It  shouldn't, e.g.   astroquery and pyvo don't have  one and twine is quite happy with releases :)

Reply:
Ah, good. I wanted to remove the email from STScI packages years ago but ended up not doing it, must had been for other reasons then. Just making sure.

Reply:
I removed the `author_email` and updated the `author` to be consistent with the LICENSE.txt PR.